chat.rocket.RocketChat.json

```diff
@@ -0,0 +1,66 @@
+ {
+    "app-id": "chat.rocket.RocketChat",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "/app/Rocket.Chat+/rocketchat",
+    "separate-locales": false,
+    "rename-desktop-file": "rocketchat.desktop",
+    "rename-icon": "rocketchat",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "rocketchat-desktop",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x rocketchat_*.deb",
+                "rm -f rocketchat_*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "install -D rocketchat.64x64.png /app/share/icons/hicolor/64x64/apps/rocketchat.png",
+                "install -D rocketchat.128x128.png /app/share/icons/hicolor/128x128/apps/rocketchat.png",
+                "chmod -R a-s,go+rX,go-w /app/Rocket.Chat+",
+                "sed -i 's|Exec=.*|Exec=/app/Rocket.Chat+/rocketchat %U|' /app/share/applications/rocketchat.desktop",
+                "install -Dm644 chat.rocket.RocketChat.appdata.xml /app/share/appdata/chat.rocket.RocketChat.appdata.xml"
+            ],
+            "sources": [
```

Comment:
Please add tarball with source code too (so it gets exported to Sources extension).